ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx

```diff
@@ -69,7 +69,9 @@ interface Props {
 const AttributeRow = (attr: {title: string; value: any}) => (
     <div className='row white-box__details-row' key={attr.title}>
         <div className='columns small-4'>{attr.title}</div>
-        <div className='columns columns--narrower-height small-8'>{attr.value}</div>
+        <div className='columns columns--narrower-height small-8' style={{whiteSpace: 'pre-line'}}>
+            {attr.value}
+        </div>
```

Comment:
`whiteSpace: 'pre-line'` require to show break line in inputs.

![image](https://user-images.githubusercontent.com/915731/152641010-faae653d-d195-4603-bfa4-855e4d8ba46b.png)